rtc/configuration.md

```diff
@@ -304,46 +359,65 @@ This option allows applications to show when a user enters the RTC session. When
 
+If a user details provider is not configured, this will be an empty object.
+
 #### Client ID
 
+| Field | Type |
+|-------|:----:|
+| `clientId` | `string` |
+
 This is a unique identifier, generated by the RTC protocol, that can be used to differentiate between the same user connecting multiple times.
 
 #### Caret Number
 
-This number corresponds to one of [the 8 colours defined in TinyMCE CSS](https://github.com/tinymce/tinymce/blob/master/modules/oxide/src/less/theme/content/rtc/rtc.less#L1-L8). TinyMCE supports 8 distinct caret colors. If more than 8 clients connect to a session, the numbers will be reused.
+| Field | Type |
+|-------|:----:|
+| `caretNumber` | `integer` |
 
+This will be a number between 1 and 8, corresponding to one of [the 8 colours defined in TinyMCE CSS](https://github.com/tinymce/tinymce/blob/master/modules/oxide/src/less/theme/content/rtc/rtc.less#L1-L8). TinyMCE supports 8 distinct caret colors. If more than 8 clients connect to a session, the numbers will be reused.
 
 A custom skin is required to change these colours, and no more than 8 are supported in this release. For information on creating a custom skin, see: [Customizing the Editor UI]({{site.baseurl}}/general-configuration-guide/customize-ui/).
 
 #### Client information
 
-This is a copy of the [`rtc_client_info`](#rtc_client_info) data from the remote user's editor configuration.
+| Field | Type |
+|-------|:----:|
+| `clientInfo` | `object` |
 
-> **Caution**: {{site.productname}} cannot guarantee the accuracy of client information data, which comes from a remote object. {{site.companyname}} recommends only using the client information data for status flags. To obtain authentic client information, use the [`rtc_user_details_provider`](#rtc_user_details_provider) data returned through the `userDetails` field.
+This is a copy of the [`rtc_client_info`](#rtc_client_info) data from the remote user's editor configuration.
 
-**Type:** `Function`
+If none was configured, this will be an empty object.
 
-**Required:** no
+> **Caution**: {{site.productname}} cannot guarantee the accuracy of data which comes from a remote object. {{site.companyname}} recommends only using the client information data for status flags. To obtain authentic client information, use the [`rtc_user_details_provider`](#rtc_user_details_provider) data returned through the `userDetails` field.
 
-#### Input fields for `rtc_client_connected`
+#### Example of client connected function
 
-| Field | Type | Description |
-|-------|:----:|-------------|
-| `userId` | `string` | The user's unique ID. |
-| `userDetails` | `object` | User details object. If none is configured, this will be an empty object. |
-| `clientId` | `string` | The unique ID for the new client connection |
-| `caretNumber` | `integer` | The user's caret number (1-8). |
-| `clientInfo` | `object` | Additional client information. If none was configured, this will be an empty object. |
+See below.
```

Comment:
This is a bit dangerous if content moves and changes

Reply:
I don't like many things about this section, but at least it's easier to follow now. @tylerkelly13 has said he'd look at ways to improve it 🤔